templates/ammos-cubs-ait.template.yaml

```diff
@@ -217,6 +217,11 @@ Resources:
           git checkout 2.3.1
           pip install .
 
+          git clone https://github.com/NASA-AMMOS/AIT-DSN.git $PROJECT_HOME/AIT-DSN
+          cd $PROJECT_HOME/AIT-DSN
+          git checkout 2.0.0
+          pip install .
+          
```

Comment:
good job duplicating the addition here from bootstrap.sh - technically that sh file isn't used yet but we will want to swap over to it in the next round of improvements